0_intro_to_ml_using_jax.ipynb

```diff
@@ -0,0 +1,3758 @@
+{
```

Comment:
<p>I revieved an error (jax or jnp not defined or something like that) after swapping to a TPU and running this cell. So, I hit "run all" to make sure everything was imported properly and then my runtime unexpectedly crashed (repeatedly) when using jax to plot sin(x). I think the <code>jax.tools.colab_tpu.setup_tpu()</code> call must not be working properly because I seem to be getting CUDA errors that make my Kernal restart:</p><p><code>external/org_tensorflow/tensorflow/stream_executor/cuda/cuda_driver.cc:265] failed call to cuInit: CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected</code></p><p>Also, if we are swapping runtimes here, we might need to let them know they need to re-run the first few cells (or just put that code here), so that libraries are imported.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='126'/>